src/components/WriteBar/WriteBar.css

```
@@ -61,8 +77,11 @@
 
 .WriteBar--ios .WriteBar__formIn {
   box-sizing: border-box;
-  background-color: var(--input_background);
-  border: 1px solid var(--input_border);
+  background-color: var(
+    --input_background,
+    var(--vkui--color_field_background)
+  );
+  border: 1px solid var(--input_border, var(--vkui--color_field_border_alpha));
```

```suggestion
  border: var(--thin-border) solid var(--input_border, var(--vkui--color_field_border_alpha));
```

`--thin-border` учитывает разрешение экрана